src/Ocean/Model/HydrostaticBoussinesqModel.jl

```diff
@@ -313,7 +314,8 @@ end
 end
 
 @inline function ocean_boundary_state!(::HBModel, ::CoastlineNoSlip,
-                                       ::Union{Rusanov, CentralGradPenalty},
+                                       ::Union{Rusanov,
+                                               CentralNumericalFluxGradient},
                                        Q⁺, A⁺, n⁻, Q⁻, A⁻, t)
   Q⁺.u = -Q⁻.u
```

Comment:
Do these boundary state functions need to be updated to reflect the fact that we are now calculating `F^{*}` instead of `F^{*} - F^{-}`?

Reply:
No. The state currently specified should be fine. We're just moving `F^{-}` out of the flux function into the kernel. So this should not effect any current functionality or results. The hope is that this will simplify more general bcs later, since the user doesn't need to figure out what the plus side should be if that's tricky (e.g., you do not need to invert a complicated boundary condition).